Sources/VaporPostgreSQL/Provider.swift

```diff
@@ -25,11 +25,7 @@ public final class Provider: Vapor.Provider {
 
     public convenience init(config: Config) throws {
         guard let postgresql = config["postgresql"]?.object else {
-            // remove this once `missing(file: String)` case is added to ConfigError
-            struct NoMySQLConfig: Error, CustomStringConvertible {
-                var description: String { return "No `mysql.json` config file found" }
-            }
-            throw  ConfigError.unspecified(NoMySQLConfig())
+            throw ConfigError.missingFile("No `postgresql.json` config file found")
```

Comment:
Write just `ConfigError.missingFie("postgresql")` and the message will be created automatically